src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -318,15 +318,9 @@ private DueDateStrategy determineStrategy(
       }
     }
     else if(isFixed(loansPolicy)) {
-      if(isRenewal) {
-        if (isRenewalWithHoldRequest) {
-          return new RollingRenewalDueDateStrategy(getId(), getName(), systemDate,
-            "SYSTEM_DATE", getAlternateRenewalLoanPeriodForHolds(),
-            new NoFixedDueDateSchedules(), this::loanPolicyValidationError);
-        } else {
-          return new FixedScheduleRenewalDueDateStrategy(getId(), getName(),
-            getRenewalFixedDueDateSchedules(), systemDate, this::loanPolicyValidationError);
-        }
+      if (isRenewal) {
```

Comment:
@marcjohnson-kint, I think we will block creating such policies in storage module. Do  you think we need an error anyway?

Reply:
Given that the interface for storage is not changing, it cannot be guaranteed that any implementation will block these combinations.

I think it could be useful to provide an error message in situations where this has occurred, for easier diagnosis by a user.

Reply:
Done.